setup/config_geoips

```diff
@@ -43,10 +43,20 @@ export CARTOPY_DATA_DIR=$GEOIPS_DEPENDENCIES_DIR/CARTOPY_DATA_DIR
 export PATH=$GEOIPS_DEPENDENCIES_DIR/bin:$PATH
 
 ############################################################################
-# Everything after this point are optional convenience functions.  Must set 
+# Everything after this point are optional convenience functions.  Must set
 # CURRENT_CONFIGPATH before sourcing.
 ############################################################################
 # Convenience variables
 export GEOIPS=$GEOIPS_PACKAGES_DIR/geoips
 export GEOIPS_SETUP=$GEOIPS/setup
 source $GEOIPS_SETUP/bash_setup/gitall
+
+# In order to use the MTG FCI reader, the HDF5_PLUGIN_PATH is a required environment variable
```

Comment:
Should this go in the docs?

Reply:
Umm, probably, unless someone else figures out a better way to handle this.... I'm guessing it will be fixed upstream at some point...

Reply:
Should we create an issue for this?  We were hoping it would be very very temporary, and it will be fixed upstream soon.

Reply:
^ Leaving for now